packages/trader/src/App/Components/Elements/ContractAudit/contract-details.tsx

```diff
@@ -44,14 +67,14 @@ const ContractDetails = ({ contract_end_time, contract_info, duration, duration_
         display_number_of_contracts,
     } = contract_info;
 
-    const is_profit = profit >= 0;
+    const is_profit = Number(profit) >= 0;
     const cancellation_price = getCancellationPrice(contract_info);
     const show_barrier = !is_vanilla && !isAccumulatorContract(contract_type) && !isSmartTraderContract(contract_type);
-    const show_duration = !isAccumulatorContract(contract_type) || !isNaN(contract_end_time);
+    const show_duration = !isAccumulatorContract(contract_type) || !isNaN(Number(contract_end_time));
```

Comment:
I have a bit of a reservation about this. If `contract_end_time` is a number or a string representation of a number ('2'). 

`!isNaN(Number(contract_end_time)` this expression will always be `true`

it'll be false if contract_end_time is non numeric string like 'hello', only then it will return false.

Is this what we want? Previously `!isNaN(Number(contract_end_time)` this expression will return `false` if `contract_end_time` is anything other than a number.

Reply:
`contract_end_time` might be a number or undefined: it comes from `<ContractDrawer /> `where a function `getEndTime` returns `contract_end_time` as a prop. `getEndTime` is in TS, so I checked that it returns `number | undefined` as `contract_end_time`.
If `contract_end_time` is `undefined`, that `Number(undefined)` is `NaN`. Not sure if it break the code

Reply:
gotcha. i think it'll work fine